packages/components/select/src/select.vue

```diff
@@ -554,8 +557,15 @@ export default defineComponent({
       ),
     ])
 
+    // if in form and use statusIcon, the width of the icon needs to be subtracted, fix #13526
     const selectTagsStyle = computed(() => ({
-      maxWidth: `${unref(inputWidth) - 32}px`,
+      maxWidth: `${
+        unref(inputWidth) -
+        32 -
+        (needStatusIcon.value && validateState.value && validateIcon.value
```

Comment:
The judgment here is placed in `useSelect`, and then `useSelect` only needs to export a value. Would it be better to only import a value here? 🤔

Reply:
That would indeed be better, that other style of judgment should be placed in `useSelect`?
I also find some `inline-style` in `select.vue` like this:
https://github.com/element-plus/element-plus/blob/499934c2499276d7662cdb9f939f6f441a37f2f5/packages/components/select/src/select.vue#L206-L213
Does it has a refactor plan?

Reply:
This pr is best to only modify the relevant styles. 

Regarding other style refactorings, there are currently not enough human resources, so there are no concrete plans. Feel free to create a pr if you're willing to refactor.  ❤️

Reply:
ok, I will change it later.
I will make a new pr to refactor computed style after this pr is merged.